go.mod

```diff
@@ -5,24 +5,25 @@ go 1.12
 require (
 	github.com/bitnami-labs/flagenv v0.0.0-20190607135054-a87af7a1d6fc
 	github.com/bitnami-labs/pflagenv v0.0.0-20190702160147-b4d9f048d98f
+	github.com/evanphx/json-patch v4.2.0+incompatible // indirect
 	github.com/gomodule/redigo v2.0.0+incompatible // indirect
 	github.com/google/go-cmp v0.4.0
 	github.com/google/gofuzz v1.0.0
 	github.com/google/renameio v0.1.0
 	github.com/googleapis/gnostic v0.0.0-20171211024024-933c109c13ce // indirect
 	github.com/mattn/go-isatty v0.0.10
 	github.com/mkmik/multierror v0.3.0
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/gomega v1.7.0
 	github.com/prometheus/client_golang v1.9.0
 	github.com/spf13/pflag v1.0.5
 	github.com/throttled/throttled v2.2.2+incompatible
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/text v0.3.3 // indirect
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.16.8
-	k8s.io/apimachinery v0.16.8
-	k8s.io/client-go v0.16.8
+	k8s.io/api v0.17.11
+	k8s.io/apimachinery v0.17.11
+	k8s.io/client-go v0.17.11
```

Comment:
This is just used for the integration tests, right? I mean, this isn't used by `kubeseal` nor the controller. Am I right? I  just want to double-check that we're not including changes that can affect backwards compatibility

Reply:
hi @juan131 
client-go is used in:
- Controller
- Kubeseal
- Integration tests
The changes are working right now with 1.16.13, 1.17.11 and 1.20.4. So, in my opinion, I think that we are not breaking the backwards compatibility.